homeassistant/components/microsoft_face.py

```diff
@@ -49,6 +49,7 @@
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
         vol.Required(CONF_API_KEY): cv.string,
+        vol.Optional(CONF_SERVER_LOC, default="westus"): cv.string,
```

Comment:
API_ENDPOINT seems to refer the full endpoint URL. 
AZURE_REGION or SERVER_REGION might be more appropriate.